.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test Coverage
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - release/**
```

Comment:
We don't really have release branches here. We just tag whatever commit is ready for release on master.